app/assets/stylesheets/govuk_publishing_components/components/_organisation-logo.scss

```diff
@@ -82,6 +82,21 @@
   }
 }
 
+.gem-c-organisation-logo--inverse {
+  .gem-c-organisation-logo__container {
+    border-color: govuk-colour("white");
+    @include govuk-link-style-inverse;
+  }
+
+  .gem-c-organisation-logo__crest--eo {
+    filter: brightness(0) invert(1);
+
+    &:focus {
+      filter: none;
+    }
+  }
+}
```

Comment:
Is there a reason this class is on `gem-c-organisation-logo__crest--eo` instead of just `gem-c-organisation-logo__crest`? Currently, this would mean the variant only works with the `eo` crest.

Reply:
Oops good spot - I'll change that